src/utils/bigNumber.ts

```diff
@@ -0,0 +1,8 @@
+import type { BigNumberish } from '@ethersproject/bignumber';
+import { commify, formatUnits, parseUnits } from '@ethersproject/units';
+
+export const parseToFormattedNumber = (value: string | BigNumberish, unit?: BigNumberish) => {
```

Comment:
I think you forgot to pass the unit it could be hardcoded also, to remove the need to pass every time.

Also, the value looks to be formatting wrong.
`10000000` -> `10,000.000`
`100000` -> `100.000`
`1000` -> `1.000`
`100 -> `0.100`

Reply:
ok @luizstacio thanks for explaining, I'll push a fix